src/System.ServiceModel.Primitives/ref/System.ServiceModel.Primitives.cs

```diff
@@ -1355,6 +1394,7 @@ public sealed partial class ClientOperation
         public ClientOperation(System.ServiceModel.Dispatcher.ClientRuntime parent, string name, string action) { }
         public ClientOperation(System.ServiceModel.Dispatcher.ClientRuntime parent, string name, string action, string replyAction) { }
         public string Action { get { return default(string); } }
+        public System.Collections.Generic.SynchronizedCollection<FaultContractInfo> FaultContractInfos { get { return default(System.Collections.Generic.SynchronizedCollection<FaultContractInfo>); } }
```

Comment:
This is to be consistent. All types such as System.TimeSpan are fully qualified. There is no Using statement in this file.